kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Logs.java

```diff
@@ -172,8 +173,15 @@ public static void archiveLog(String testClass, String testMethod, File log) thr
         }
         Path destDir = getLogsDir(testClass, testMethod);
         Files.createDirectories(destDir);
-        String filename = log.getName();
-        Files.copy(log.toPath(), destDir.resolve(filename), REPLACE_EXISTING);
+        try (Stream<Path> walk = Files.walk(appLogsDir)) {
+            walk.skip(1).forEach(path -> { // skip root directory
+                try {
+                    Files.copy(path, destDir.resolve(appLogsDir.relativize(path)), REPLACE_EXISTING);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
is this print instead of log on purpose?

Reply:
You are right, I thought I would change it when I did it, but forgot. Now it's fixed.